bin/getmaxmind.js

```
@@ -0,0 +1,47 @@
+'use strict';
+const fs = require('fs');
+const http = require('http');
+const targz = require('targz');
+
+const URL = 'http://geolite.maxmind.com/download/geoip/database/GeoLite2-City.tar.gz';
+const DB_DIR = `${__dirname}/../db`;
+const GZ_FILE = `${DB_DIR}/GeoLite2-City.tar.gz`;
+const DB_FILE = `${DB_DIR}/GeoLite2-City.mmdb`;
+
+// check for existing file
+try {
+  fs.mkdirSync(DB_DIR);
```

This eats any error, yes?
Maybe instead of a try catch:
```
if (!fs.existsSync(DB_DIR)){
    fs.mkdirSync(DB_DIR);
}
```